### PR TITLE
UI for to show memory pool for running queries.

### PR DESCRIPTION
**Pull request by idemura**
Tested manually.

### CODE REVIEW

**Comment by petroav** [comment]
The first "for" in your commit message can be removed. Also, don't end the subject line of your commit message with a period. For more details on what the expected commit message format is, see http://chris.beams.io/posts/git-commit/.

**Comment by haozhun** [comment]
Please fix the checkstyle failures. (You can click on the "cross" icon next to your commit to see them in Travis)

**Comment by idemura** [comment]
Done.

File: presto-main/src/main/resources/webapp/assets/query-list.js [hunk 1]
```diff
@@ -78,6 +78,9 @@
                 else {
                     summary = (progress == 0 ? summary : summary + " (" + progress + "%" + ")");
                 }
+                if (query.memoryPool == "reserved") {
```
Comment (electrum) [inline presto-main/src/main/resources/webapp/assets/query-list.js]: Why hard code this and not display whatever value is returned?

Comment (idemura) [inline presto-main/src/main/resources/webapp/assets/query-list.js]: It sounded as "mark query running in reserved pool". We considered marking with color. Later with text. I think it good to put text unconditionally. Pure visually, it wouldn't be highlighted.

**Comment by haozhun** [comment]
Merged #6068

**Pull request closed without merging**